### ISSUE TITLE
Hyperparameter Optimization

### ISSUE THREAD

**Issue by xown3197** [sorachin/AI_Team10#12]
- Hyperparameter Optimization 는 learning rate, batch, epoch, 층의 갯수, 정규화 정도, 필터 크기 등 학습이 아닌 사람이 정하는 변수를 칭합니다.
- Hyperparameter Tuning은 일정 수준의 모델에서는 성능의 우세를 결정하는 중요한 요소입니다.
- 최적화를 하는 방법으로는 bayesian optimization, genetic algorithms 등 사용이 되고 각광 받는 방법으로는 random search가 있습니다. 
- 단, 너무 과한 Hyperparameter Optimization는 Over fitting을 발생시킬 수 있습니다.

Ps. 기회가 된다면 Hyperparameter Optimization도 적용해볼 생각입니다.
